packages/components/src/components/contract-card/contract-card-items/contract-card-header.tsx

```diff
@@ -71,12 +71,14 @@ const ContractCardHeader = ({
     const is_mobile = isMobile();
     const is_turbos = isTurbosContract(contract_type);
     const is_lookbacks = isLookBacksContract(contract_type);
+    const is_multipliers = !!multiplier && !is_lookbacks;
```

Comment:
im not understanding this condition. Whats the relation between multiplier and lookbacks

Reply:
Both have multiplier (e.g. x100, x200 ). So we need to determine somehow if this contract is Multipliers or LookBacks

Reply:
In this particular place I'm checking if this contract has multiplier and it's not Lookbacks ==> then it's Multipliers